lib/existing-eks-opensource-observability-pattern/index.ts

```diff
@@ -7,6 +7,8 @@ import { ObservabilityBuilder } from '@aws-quickstart/eks-blueprints';
 import * as cdk from "aws-cdk-lib";
 import * as eks from 'aws-cdk-lib/aws-eks';
 import * as fs from 'fs';
+import { IstioIngressGatewayHelmAddon } from './istio/istioIngressGatewayAddon';
```

Comment:
This was never used `IstioIngressGatewayHelmAddon` and `IstioCniHelmAddon`. Please add the logic for istio from new OSS pattern code.